### PR TITLE
[Docs] Updating helm README file for issue #1545

### PR DESCRIPTION
**Pull request by gnewson**
This PR relates to https://github.com/chainguard-images/images/issues/1545

The only part of the checklist that is appropriate is regarding the documentation and is included below. I hope this is ok as I can't provide any sensible choices for the other 9 points.

Documentation - README
[ x] A README file has been provided and it follows the README template.
The README has been updated to reflect the template. It could still do with some usage information but hopefully this is an improvement on what's currently there.

@ltagliaferri you mentioned tagging you in for review.

If there are further changes that are desirable please let me know.

### CODE REVIEW

**Comment by gnewson** [comment]
Hi @erikaheidi,
I see that the presubmit-readme job failed with the following:

> images/helm/README.md is out-of-date.
> Error: check failed
> Found a total of 1 issue(s). Please run the following at the root of this repo to fix: "monopod readme"

Is that something I need to address in my local repository?
Thanks.

**Comment by erikaheidi** [comment]
> Hi @erikaheidi, I see that the presubmit-readme job failed with the following:
> 
> > images/helm/README.md is out-of-date.
> > Error: check failed
> > Found a total of 1 issue(s). Please run the following at the root of this repo to fix: "monopod readme"
> 
> Is that something I need to address in my local repository? Thanks.

Hi @gnewson, we have a tool called `monopod` that adds some metadata to new READMEs and also works as a linter. This tool is included in the repo. To get it installed:

```
cd monopod
go install
```
Then, you can run it from the root of the repository:

```
monopod readme
```

And this should fix the issue for you.

Apologies as this was not included in the docs, I will make a PR to add this information to our best practices doc.

File: images/helm/README.md [hunk 1]
```diff
@@ -12,12 +12,27 @@
...
-## Get It!
+- [Documentation](https://edu.chainguard.dev/chainguard/chainguard-images/reference/helm)
+- [Provenance Information](https://edu.chainguard.dev/chainguard/chainguard-images/reference/helm/provenance_info/)
 
-The image is available on `cgr.dev`:
+## Image Variants
+
+Our `latest` tags use the most recent build of the [Wolfi helm](https://github.com/wolfi-dev/os/blob/main/helm.yaml) package. The following tagged variants are available without authentication:
+
+- `latest`: This is a distroless image for running helm to install packages to a kubernetes cluster. It does not include Composer or busybox, so no shell will be available.
```
Comment (erikaheidi) [inline images/helm/README.md:24]:
```suggestion
- `latest`: This is a distroless image for running helm to install packages to a kubernetes cluster. It does not include `apk-tools` or `bash`, so no shell will be available.
```

File: images/helm/README.md [hunk 2]
```diff
@@ -12,12 +12,27 @@
...
+- `latest-dev`: This is a development / builder image that includes Composer, apk-tools, and busybox. This variant allows you to customize your final image with additional Wolfi packages.
```
Comment (erikaheidi) [inline images/helm/README.md:25]:
```suggestion
- `latest-dev`: This is a development / builder image that includes `bash`, `apk-tools`, and `busybox`. This variant allows you to customize your final image with additional Wolfi packages.
```

**Review by erikaheidi — CHANGES_REQUESTED** [review]
Thanks for contributing with a README for helm! There are just a couple changes to remove mentions to Composer, since that is a package only present in PHP images. After that we can merge 🙏🏼

**Comment by gnewson** [comment]
@erikaheidi the issue that monopod picked up was that I'd removed a '*' from the end of a line that was generated by the template.  I didn't see the significance but I may be missing something.  If I'm not, is it worth a small PR to change the template:
https://github.com/chainguard-images/images/blob/c6d9193f223bfe2fce743e7c8756696d94d4c204/monopod/pkg/commands/readme.go#L73C150-L73C150

**Review by erikaheidi — APPROVED** [review]

**Pull request merged**